### ISSUE TITLE
Pipeline fails during composer install

### ISSUE THREAD

**Issue by torhoehn** [captainhook-git/plugin-composer#13]
In our GitLab Pipeline the plugin breaks a `composer install`, because no git directory exists. Would it be possible to skip the installation of the hooks in this case? Maybe using a CI/CD variable?

```
CaptainHook Composer Plugin
Installation failed, reverting ./composer.json to its original content.
In ComposerPlugin.php line 274:
                                                                               
  Shiver me timbers! CaptainHook could not install yer git hooks! (git directory not found)
```

**Comment by heiglandreas** [comment]
@sebastianfeldmann: Would checking for an existing `.git` folder be something that The Cap'n should do? Or would we skip installing The Ca'n in the first place when no `.git` folder is found?

Or should we check for the usual suspects like `CI`-environment variables to not install The Cap'n in the first place? That might make more sense perhaps...

**Comment by sebastianfeldmann** [comment]
I just released version 5.2.4 with the fix

Big thanks to @heiglandreas :)

**Comment by torhoehn** [comment]
Thanks to both of you!
